natlas-server/app/admin/routes.py

```diff
@@ -90,32 +89,26 @@ def deleteUser(id):
 
 
 @bp.route("/users/<int:id>/toggle", methods=["POST"])
-@is_authenticated
+@login_required
 @is_admin
-def toggleUser(id):
+def toggle_user(id):
```

Comment:
Toggle operations are unsafe because they introduce you to race conditions. If another person clicks toggle after you've loaded the page, you'll accidentally double toggle the user.

Reply:
Captured in #320